package.json

```diff
@@ -8,12 +8,12 @@
     "react-dom": "^16.0.0",
     "geojs": "0.12.1",
     "hammerjs": "^2.0.8",
-    "reconcile.js": "^1.3.2",
```

Comment:
for what purpose we were using the reconcile?

Reply:
When this project is not a react application so I was using it to help DOM updating.